hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableNftStore.java

```diff
@@ -101,4 +101,26 @@ public long sizeOfState() {
     public Set<UniqueTokenId> modifiedNfts() {
         return nftState.modifiedKeys();
     }
+
+    /**
+     * Removes the {@link Nft} with the given serial number
+     *
+     * @param serialNum - the combined unique ID of the NFT to remove
+     */
+    public void remove(final @NonNull UniqueTokenId serialNum) {
```

Comment:
Nit: I think our coding standard is `@NonNull final`, but if Spotless isn't complaining, then neither will I :)

Reply:
Ugh you're right. I'll nab this in my next PR